src/reactionsystem.jl

```diff
@@ -54,6 +54,17 @@ end
 # variables
 drop_dynamics(s) = isconstant(s) || isbc(s) || (!isspecies(s))
 
+### Other Metadata Fields ###
+
+# Denotes that a parameter controls the scaling of noise in the CLE.
+struct NoiseScalingParameter end
+Symbolics.option_to_metadata_type(::Val{:noisescalingparameter}) = NoiseScalingParameter
+
+isnoisescalingparameter(s::Num) = isnoisescalingparameter(MT.value(s))
```

Comment:
```julia
isnoisescalingparameter
```
is sufficiently long that I'd suggest adding underscores. It is hard to read as is.

Reply:
Just took the standard of other metadata, but agree. Should I change:
`isnoisescalingparameter` to `is_noise_scaling_parameter`
and
`noisescalingparameter` to `noise_scaling_parameter` (where this is used, e.g. when declaring metadata)
?

Reply:
Sure, sounds good.